### PR TITLE
blogs 1 year anniversary for Pyrsia

### PR DESCRIPTION
**Pull request by betarelease**

### CODE REVIEW

File: blog/2022-12-30-pyrsia-anniversary-recap.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## Security taken for granted
+
+Before the SolarWinds attack and widespread news attention it received security was taken for granted. Open source software continues to be build by individuals using trust metrics that work in real life. These trust mechanisms like `sharing commit access/publish rights, with other contributors was a widespread community practice` listed in [Rise of few-maintainer projects](https://increment.com/open-source/the-rise-of-few-maintainer-projects/) create a weak infrastructure that all software projects rely on. This worked as long as the awareness as well as impact was limited. With growing reliance on software from keeping the lights on to keeping us healthy and everything in between, this model of `trust` does not scale.
```
Comment (fishseabowl) [inline blog/2022-12-30-pyrsia-anniversary-recap.md:21]: typo
```suggestion
Before the SolarWinds attack and widespread news attention it received security was taken for granted. Open source software continues to be built by individuals using trust metrics that work in real life. These trust mechanisms like `sharing commit access/publish rights, with other contributors was a widespread community practice` listed in [Rise of few-maintainer projects](https://increment.com/open-source/the-rise-of-few-maintainer-projects/) create a weak infrastructure that all software projects rely on. This worked as long as the awareness as well as impact was limited. With growing reliance on software from keeping the lights on to keeping us healthy and everything in between, this model of `trust` does not scale.
```

File: blog/2022-12-30-pyrsia-anniversary-recap.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Pyrsia started to challenge this status quo and provide a better, available, transparent supply chain for your software.
+
+## Navigating the technology space
+
+In the initial days the mission was getting clearer while we were researching and prototyping Everything!. Everything included - networking technologies, blockchain ledgers and technologies, programming in Rust and putting it all together. As we did this we were able to identify goals that we could incremental build towards. We started with the aim of building a working version of the decentralized network that would let users download packages from peers. With this as our minimal requirement we experimented with different tools that could help us deliver.
```
Comment (fishseabowl) [inline blog/2022-12-30-pyrsia-anniversary-recap.md:27]: typo
```suggestion
In the initial days the mission was getting clearer while we were researching and prototyping Everything!. Everything included - networking technologies, blockchain ledgers and technologies, programming in Rust and putting it all together. As we did this we were able to identify goals that we could incrementally build towards. We started with the aim of building a working version of the decentralized network that would let users download packages from peers. With this as our minimal requirement we experimented with different tools that could help us deliver.
```

File: blog/2022-12-30-pyrsia-anniversary-recap.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+We were able to resolve some of those errors slowly by moving to a more stable and supported tokio.rs library for all our async processing. Tieing it together libp2p made the memory leak go away and provide lot more stability that we had in the previous version.
+
+When we had it all working we released a [Demo Pyrsia network](https://github.com/pyrsia/pyrsia/releases/tag/v0.1.0).
+
+Releasing this working version where documentation matched behaviour and consistenly worked on all our machines and the cloud was our first big win! This came after tireless efforts at learning the language, the ecosystem, and the underlying technology while iteratively writing code.
```
Comment (fishseabowl) [inline blog/2022-12-30-pyrsia-anniversary-recap.md:62]: typo
```suggestion
Releasing this working version where documentation matched behaviour and consistently worked on all our machines and the cloud was our first big win! This came after tireless efforts at learning the language, the ecosystem, and the underlying technology while iteratively writing code.
```

File: blog/2022-12-30-pyrsia-anniversary-recap.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+[v0.2.2 - libp2p upgrade, rust toolchain, clap update for CLI](https://github.com/pyrsia/pyrsia/releases/tag/v0.2.2)
+
+We are getting into a process where every release we are able to pick and prioritize our goals and measure our progress towards them during the 2 sprints that follow. We hope to continue this to be able to make small, incremental improvements to what Pyrsia offers.
+
+## Looking into the next year
+
+On the back of the December release we are getting ready to release v0.2.3 in January 2023. Following that we will continue making a dent in our technical debt and continue progressing towards our next major milestone of releasing Java support by version v0.3.0. Along the way we have a few other significant improvements lined up eg. a scalable build service, wider Authority node network that runs on multiple clouds, an API that will encourage integration with language and package ecosystems and so on.
+
+I can't wait to get started on 2023 and showing off new features to our community.
```
Comment (fishseabowl) [inline blog/2022-12-30-pyrsia-anniversary-recap.md:97]: typo
```suggestion
I can't wait to get started in 2023 and show off new features to our community.
```

**Review by fishseabowl — APPROVED** [review]
Good work!

**Review by dasmanas — APPROVED** [review]

**Pull request merged**